app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="<%= object.classname %>" id="mychart"></div>
```

Comment:
You could reduce interpolation here with the content_tag helper:
```suggestion
<%= content_tag(:div, "", class: object.classname, id: "mychart") %>
```
I personally find it more readable in this case.